core/sql/langman/LmRoutineJavaObj.cpp

```diff
@@ -192,6 +192,7 @@ LmResult LmRoutineJavaObj::invokeRoutineMethod(
           (*emitRowPtr_)(NULL,0,&qs);
         }
     }
+  jni->DeleteLocalRef(jniResult);
```

Comment:
@selvaganesang, we are not using the local ref across two invocations. We are using it, however, after returning from this method, before we invoke the method again.